web/src/app/modules/views/main/authentication/modules/login/components/login.component.html

```diff
@@ -61,9 +61,9 @@ <h1 class="card-title text-center mb-4 mt-1">{{'login' | translate}}</h1>
                     <i class="fa fa-sign-in"></i>&nbsp;{{'login' | translate}}
                 </ng-container>
             </button>
-            <span id="login-language-dropdown" class="pull-right">
+            <button class="btn btn-sm btn-outline-primary pull-right"> <span id="login-language-dropdown" class="pull-right">Language

```

Comment:
Looks like you have mixed together two features in this PR.

Reply:
Hi Eriks,

Thanks for your comment!

Yes, I have committed twice and sent the Pull Request once for my assignment.

Best Regards,
Maryam Rezaei,